\input{../utils/preamble.tex}
\input{../utils/macros.tex}
\bibliography{../bib/references}

\title[History of Lineages]{History of Lineages}
\subtitle{Chapter 11}

\author[J.\ Oaks]{
    Jamie Oaks\inst{1}
}
\institute[University of Washington]{
    \inst{1}%
        Kincaid Hall 524 \hspace{1em} \href{mailto:joaks1@gmail.com}{\texttt{joaks1@gmail.com}}
}

% \date{\today}
\date{April 11, 2014}

\begin{document}

% \maketitle
\begin{frame}
    \begin{columns}[c]
        \column{.5\textwidth}
            \maketitle
        \column{.5\textwidth}
            \begin{figure}
                \begin{center}
                \includegraphics[width=\textwidth]{../images/darwin-tol-copyright-boris-kulikov-2007.jpg}
                \caption{\tiny \copyright~2007 Boris Kulikov}
                \end{center}
            \end{figure}
    \end{columns}
\end{frame}

\begin{frame}
    \frametitle{Acknowledgements}
\end{frame}

\begin{frame}
    \frametitle{Bergmann's Rule}
    \begin{columns}[c]
        \column{.5\textwidth}
            \begin{onlyenv}<1->
            \begin{center}
            \begin{tikzpicture}
            [scale=0.35,auto=left,every node/.style={circle}]%,fill=blue!20}]
              \draw [<->,thick] (0,10) -- (0,0) -- (14,0);
              \node [label=below: {\sffamily Latitude}](xl) at (7.5,0.5) {};
              \node [label=left: {\sffamily Mass}](yl) at (0,5) {};
              \node [inode, label={[label distance=-5pt]0:\small\sffamily\bfseries A}](a) at (1,1) {};
              \node [inode, label={[label distance=-5pt]0:\small\sffamily\bfseries B}](d) at (2.5,0.8) {};
              \node [inode, label={[label distance=-5pt]0:\small\sffamily\bfseries C}](c) at (2,2.5) {};
              \node [inode, color=blue, label={[label distance=-5pt]0:\small\sffamily\bfseries D}](e) at (7,7.5) {};
              \node [inode, color=blue, label={[label distance=-5pt]0:\small\sffamily\bfseries E}](f) at (8.1,9.1) {};
              \node [inode, color=blue, label={[label distance=-5pt]0:\small\sffamily\bfseries F}](b) at (10,10) {};
            
            \end{tikzpicture}
            \end{center}
            \end{onlyenv}
        \column{.5\textwidth}
            \begin{onlyenv}<2>
            \begin{center}
            \begin{tikzpicture}
            [scale=0.35,auto=left,every node/.style={circle}]%,fill=blue!20}]
              \node [inode, color=black, label={[label distance=-5pt]90:\small\sffamily\bfseries A}](a) at (1, 7) {};
              \node [inode, color=black, label={[label distance=-5pt]90:\small\sffamily\bfseries B}](b) at (3, 7) {};
              \node [inode, color=black, label={[label distance=-5pt]90:\small\sffamily\bfseries C}](c) at (5, 7) {};
              \node [inode, color=blue, label={[label distance=-5pt]90:\small\sffamily\bfseries D}](d) at (7, 7) {};
              \node [inode, color=blue, label={[label distance=-5pt]90:\small\sffamily\bfseries E}](e) at (9, 7) {};
              \node [inode, color=blue, label={[label distance=-5pt]90:\small\sffamily\bfseries F}](f) at (11, 7) {};

              \draw [very thick] (a) -- (3,5) -- (4,6) -- (c);
              \draw [very thick] (4,6) -- (3.5, 6.5);
              \draw [very thick] (3.5, 6.5) -- (b);
              \draw [very thick, draw=blue] (d) -- (9,5) -- (10,6) -- (f);
              \draw [very thick, draw=blue] (10,6) -- (9.5, 6.5);
              \draw [very thick, draw=blue] (9.5, 6.5) -- (e);
              \draw [very thick] (3,5) -- (6,-4) -- (7.5, 0.5);
              \draw [very thick, draw=blue] (7.5, 0.5) -- (9, 5);
              \draw [very thick] (6,-4) -- (6, -5);
            
            \end{tikzpicture}
            \end{center}
            \end{onlyenv}

            \begin{onlyenv}<3>
            \begin{center}
            \begin{tikzpicture}
            [scale=0.35,auto=left,every node/.style={circle}]%,fill=blue!20}]
              \node [inode, color=black, label={[label distance=-5pt]90:\small\sffamily\bfseries A}](a) at (1, 7) {};
              \node [inode, color=blue, label={[label distance=-5pt]90:\small\sffamily\bfseries E}](b) at (3, 7) {};
              \node [inode, color=black, label={[label distance=-5pt]90:\small\sffamily\bfseries C}](c) at (5, 7) {};
              \node [inode, color=blue, label={[label distance=-5pt]90:\small\sffamily\bfseries D}](d) at (7, 7) {};
              \node [inode, color=black, label={[label distance=-5pt]90:\small\sffamily\bfseries B}](e) at (9, 7) {};
              \node [inode, color=blue, label={[label distance=-5pt]90:\small\sffamily\bfseries F}](f) at (11, 7) {};

              \draw [very thick] (a) -- (3,5) -- (4,6) -- (c);
              \draw [very thick] (4,6) -- (3.5, 6.5);
              \draw [very thick, draw=blue] (3.5, 6.5) -- (b);
              \draw [very thick, draw=blue] (d) -- (9,5) -- (10,6) -- (f);
              \draw [very thick, draw=blue] (10,6) -- (9.5, 6.5);
              \draw [very thick] (9.5, 6.5) -- (e);
              \draw [very thick] (3,5) -- (6,-4) -- (7.5, 0.5);
              \draw [very thick, draw=blue] (7.5, 0.5) -- (9, 5);
              \draw [very thick] (6,-4) -- (6, -5);
            
            \end{tikzpicture}
            \end{center}
            \end{onlyenv}
    \end{columns}
\end{frame}

% what is phylogenetics
% why phylogenetics is important
%     nothing in biology dob quote
%     Seen in the light of evolution, biology is, perhaps, intellectually the
%     most satisfying and inspiring science. Without that light it becomes a pile
%     of sundry facts some of them interesting or curious but making no
%     meaningful picture as a whole.
%     Dobzhansky, T. (1973). Nothing in biology makes sense except in the light of evolution. The American Biology Teacher 35:125--129.
%     ... nothing in evolution makes sense except in the light of phylogeny... SSB (http://systbio.org/teachevolution.html)
%     impossible to understand biodiversity without framework
%     biogeog
%         ecological biogeog -- independent contrast stuff
%         understanding diversification across space and time

% Classification before phylogeny -- been classifying stuff since hg days -> tends to form hiearchies
% Linneaus formalized this
% Evolution gave the framework behind why hiearchies were intuitive

%TERMINOLOGY
% Tree anatomy stuff
    % branches tips nodes etc
    % monophyly poly para etc.
    % branch rotation

% character terms
    % homology (shared due to no change) / homoplasy ("same change" has more than once; shared due to changes)
        % homologous
            % plesio / apo morphy
            % synapo /symplesio
        
% History -- three schools
    % Evol systematics
        % key point: very subjective and recognized paraphyletic groups
        % was very popular and legacy remains... many paraphyletic groups (fish, reptiles)
    
    % Numerical phenetics/taxonomy
        % sought objective, quantitative methods.
        % quantitiative clustering algorithms cluster taxa based on similarity of phenotypic traits.
        % resulted in paraphyletic/polyphyletic groups due to similarity among convergence
        % math / algorithms produced are still very useful for multivariate stats

    % Phylogenetic systematics / Cladistics
        % hennigian logic (mth lec 4)
        % logical inference: premises -> rules -> statement
        % premise: we can code characters
        % taxa that share a character state must be more closely related
        % Brief example
        % Later was parsimony method
        % no logical way to account for homoplasies -- need an error model!

% Methods
    % not going to dwell on Hennig... important then... not so much now.
    % Current methods are statistical (model-based).

% Molecular phylogenetics
    % Early indirect approaches to measuring changes in DNA
        % immunological assays
        % protein electrophoresis
        % DNA-DNA hybridization
        % restriction enzymes
    % PCR and sequencing game changer

% Molecular evolution
    % DNA - 4 character states
        % redundancy of code
        % transition/transversion bias
        % very conducive to modeling
        % different rates across genome -- conducive for resolving trees at different timescales
    % Phylogenomics
    % total evidence
    % morph character mapping

% Phylogeography
    % DNA data allowed for constructing trees of individual organisms (or gene copies) within and among species
    % gene trees
    % gene copies haplotypes
    % lineage sorting
    % coalescence
    % reciprocal monophyly
    % haplotype networks
    % gene trees within species trees (div times and incongruence)
    % book highlights these as problems, but we now have models of gene trees within species trees
        % "problems" can be a benefit for estimating historical demography and geography

% ecol niche modeling.
    % means of indirectly estimating demographic changes/ distributional shifts



\end{document}

